### ISSUE TITLE
Libraries are not stored on the same webserver

### ISSUE THREAD

**Issue by devksingh4** [JoeyRubas/Schedule-Share#2]
Libraries such as bootstrap and jquery not being stored on the webserver and instead being pulled from an external CDN slows down the user's experience. 

Solution: Download the files and place them in your code bundle, and then reference the library as a local file instead of from the CDN.

**Comment by devksingh4** [comment]
@JoeyRubas @wjwoker Any progress?

**Comment by JoeyRubas** [comment]
We'll address this any other minute issues after the Node js build is completed

**Comment by JoeyRubas** [comment]
Especially because it's likely that the site will be migrating away from heroku

**Comment by devksingh4** [comment]
This issue has nothing to do with Heroku

Best,
Dev Singh

**Comment by JoeyRubas** [comment]
Sure, but it would be appropriate to address while setting up the production deployment as the current solution is certainly passible for development

**Comment by devksingh4** [comment]
No, this is a client side html change. This should be done asap.

Best,
Dev Singh
